docs/notebooks/2_getting_started.ipynb

```diff
@@ -38,10 +38,10 @@
     "\n",
```

Comment:
<p>I would create all the fake data in one shot instead of having to "pollute" other cells.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>